packages/design-system/src/scalars/components/radio-group/radio/radio.stories.tsx

```diff
@@ -0,0 +1,263 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Radio } from "./radio";
+import { RadioGroup } from "../radio-group";
+
+const meta: Meta<typeof Radio> = {
+  component: Radio,
+  decorators: [
+    (Story) => (
+      <RadioGroup name="radio-group">
+        <Story />
+      </RadioGroup>
+    ),
+  ],
+  parameters: {
+    controls: { sort: "requiredFirst" },
+  },
+  tags: ["autodocs"],
+  title: "Document Engineering/Simple Components/Radio Group/Radio",
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Radio>;
+
+export const Default: Story = {
+  argTypes: {
+    label: { control: "text" },
+    value: { control: "text" },
+  },
+  args: {
+    label: "Default",
+    value: "default",
+  },
+};
+
+export const DefaultChecked: Story = {
```

Comment:
It's possible to moving the shared argTypes (like label, value, checked, description, and readOnly) to the meta level instead of defining them in each story? This would simplify the code and reduce repetition. What do you think?

Reply:
For me this makes sense in the case of "label" and "value" which are common in all variants, but the other controls are used depending on the variant. WYF guys? @alejandrocabriales @YasielCabrera

Reply:
argTypes moved to the top (meta) level